mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -114,6 +114,10 @@ Description of the change and how users should adjust their configuration (if ne
 
 The provider will now enforce at plan time that at most one of these fields be set.
 
+### `schema` can only be represented as a JSON array with non-null elements.
+
+The provider will now enforce at plan time that `schema` can be unmarshalled into a valid JSON array with non-null elements.
```

Comment:
"unmarshalled" is a bit Go-specific - what would you think about this language instead?

```suggestion
The provider will now enforce at plan time that `schema` is a valid JSON array with non-null elements.
```

Reply:
Makes sense, thanks, updated.